README.org

```diff
@@ -234,17 +236,36 @@ var detail = {
     [uniqueId | 'id']: String, // the unique id
     title: String,             // title of the asset
     year: Number,              // year of release
-    genre: [movie.Genre],      // a free list of genre keywords
+    genres: [String],          // a free list of genre keywords
     rating: Number,            // a 0-100 rating value
-    poster: String,            // url to poster image
+    poster: String,            // poster image
     type: Provider.ItemType    // used by the browser to decide how to show the item
                                // (MOVIE, TVSHOW)
+    num_seasons: Number,       // the number of seasons available to show
+                               // only in the case of Provider.ItemType.TVSHOW
 //--- all of the above is already required by fetch(), new stuff here: ---
-    backdrop: String,          // url of the backdrop image
+    runtime: Number,           // the runtime of the item
+    backdrop: String,          // backdrop image
+    subtitle: {url: String},   // language -> url subs mapping
+    synopsis: String,          // a short description of the asset
+}
+#+END_SRC
+
+The =Provider.ItemType.MOVIE= has the following additional fields:
+#+BEGIN_SRC
+var detail = {
+//--- including all the fields of the generic detail object
     torrents: [Object],        // torrents object
-    subtitle: {String: url},   // language -> url subs mapping
```

Comment:
Seems like shows don't even have a `subtitle` field. I tested it with the `xaiki/redesign` branch.